### PR TITLE
refactor(foundations/breakpoints): proposition of new breakpoints for h…

### PR DESCRIPTION
**Pull request by marineFabien**
…armonize with LMFR

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [x] Other... Please describe:
Breaking change : Refactoring breakpoint

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

## Other information

### CODE REVIEW

**Comment by marineFabien** [comment]
http://proposenewmainbreakpoints.demo.mozaic-design-system-dtrp.appspot.com

File: src/pages/Foundations/Layout/Grid/code.md [hunk 1]
```diff
@@ -131,9 +131,10 @@ The available breakpoints modifiers are :
...
 | `ml-flexy__col--[size]` (no modifier) | from 0px and up                | mobile / all                         |
-| `ml-flexy__col--[size]@from-m`        | **m**: from **576px and up**   | fablet / small tablet and up         |
-| `ml-flexy__col--[size]@from-l`        | **l**: from **1100px and up**  | large tablets / small labtops and up |
-| `ml-flexy__col--[size]@from-xl`       | **xl**: from **1920px and up** | labtops and up                       |
+| `ml-flexy__col--[size]@from-m`        | **m**: from **680px and up**   | fablet / small tablet and up         |
+| `ml-flexy__col--[size]@from-l`        | **l**: from **1024px and up**  | landscape tablets, laptops |
+| `ml-flexy__col--[size]@from-xl`       | **xl**: from **1280px and up** | laptops (depending on settings)      |
+| `ml-flexy__col--[size]@from-xxl`       | **xxl**: from **1920px and up** | desktop                             |
 
 <br>
```
Comment (gael-boyenval) [inline src/pages/Foundations/Layout/Grid/code.md]: responsive push modifiers lacks the XXL variants

File: src/pages/Foundations/Layout/Responsive/code.md [hunk 2]
```diff
@@ -107,25 +128,24 @@ $viewports: ('s', 'm', 'l');
...
-| **1100px** | `$screen-l`        | landscape tablets, laptops                                                                                         |
-| **1280px** | `$screen-l-mini`   | laptops (depending on settings), <br>Samsung Galaxy Tab 10" (landscape), <br>Samsung Galaxy Tab 2 10" (landscape), |
-| **1440px** | `$screen-l-medium` | laptops (depending on settings), <br>Microsoft Surface Pro 3 (landscape), <br>Apple iPad Pro 12.9" (landscape)     |
-| **1680px** | `$screen-l-mega`   | laptops (depending on settings),                                                                                   |
-| -          | -                  | -                                                                                                                  |
-| **1920px** | `$screen-xl`       | desktop computers                                                                                                  |
-
+| **320px** and up  | `$screen.s-medium`   | iphone5, SE                                                                                                        |
```
Comment (gael-boyenval) [inline src/pages/Foundations/Layout/Responsive/code.md]: lacks the 0

Comment (gael-boyenval) [reply src/pages/Foundations/Layout/Responsive/code.md]: `$screen.s-medium` => `$screen-s-medium` (replace the dot with a dash everywhere)

**Comment by melanieanglardLMFR** [comment]
@gael-boyenval , everything's alright for JB and I about new breakpoints values and definitions. You can merge them. 

I don't know if @marineFabien had enought time to manage the changes you requested. If not, i can do it tomorrow to merge the PR before the end of the week.

**Comment by melanieanglardLMFR** [comment]
@gael-boyenval , every things good for me. I let you check. 

I didn't change anything about container instead of breakpoint M value. We can manage this point later.

File: src/docs/Foundations/Typography/HeroStyles/code.mdx [hunk 3]
```diff
@@ -76,12 +76,12 @@ You can import all base typography scss files by importing the `_a.all-base-typo
 
 <br />
 
-**Viewport >= L-MEDIUM (1440px) :**
+**Viewport >= L (1280px) :**
```
Comment (gael-boyenval) [inline src/docs/Foundations/Typography/HeroStyles/code.mdx]: XL ?

Comment (melanieanglardLMFR) [reply src/docs/Foundations/Typography/HeroStyles/code.mdx]: @gael-boyenval done :)

**Pull request merged**